composer.json

```diff
@@ -26,7 +26,10 @@
     "require": {
         "php": "^8.1",
         "sylius/paypal-plugin": "^1.6",
-        "sylius/sylius": "~1.13.0",
+        "sylius/core-bundle": "~1.13.0",
```

Comment:
To be honest, yes it is not required. However, I think it is good practice to have explicit composer requirements for the things that you use. Now, we can argue that the same is true for all the other sub-dependencies of sylius/core-bundle. And here I would go pragmatic and just assume that core-bundle is an aggregator for all of them. 

To sum up, I guess we would be fine with leaving core-bundle as it is. But removing it will also not change too much